lib/node_modules/@stdlib/string/prev-grapheme-cluster-break/lib/main.js

```diff
@@ -99,7 +99,9 @@ function prevGraphemeClusterBreak( str, fromIndex ) {
 
 	// Get the corresponding grapheme break and emoji properties:
 	breaks.push( breakProperty( cp ) );
-	emoji.push( emojiProperty( cp ) );
+	if (!(cp >= 0x1F3FB && cp <= 0x1F3FF)) { // Check if the character is not a skin tone modifier
```

Comment:
This is too narrow a patch. It is not just skin tones, TMK, but any modifier. The implementation more generally should be refactored to ensure that grapheme clusters comprised of multiple code points are handled correctly.

Reply:
Oh I am sorry, I got confused by seeing the original issue being raised about the colour tone.
I will try to debug and propose a generic solution to this as soon as possible.
Thank You.